app-backend/db/src/test/scala/com/azavea/rf/database/CampaignDaoSpec.scala

```diff
@@ -511,4 +511,321 @@ class CampaignDaoSpec
       )
     }
   }
+
+  test("no tasks are returned when all have been validated") {
+    check {
+      forAll(
+        (
+            userCreateBase: User.Create,
+            userCreates: List[User.Create],
+            userCreate: User.Create,
+            campaignCreate: Campaign.Create,
+            annotationProjectCreate: AnnotationProject.Create,
+            taskFeaturesCreate: Task.TaskFeatureCollectionCreate
+        ) => {
+          val copyIO = for {
+            parent <- UserDao.create(userCreate)
+            children <- userCreateBase :: userCreates traverse { u =>
+              UserDao.create(u)
+            }
+            insertedCampaign <- CampaignDao
+              .insertCampaign(
+                campaignCreate.copy(parentCampaignId = None),
+                parent
+              )
+            insertedProject <- AnnotationProjectDao
+              .insert(
+                annotationProjectCreate.copy(
+                  campaignId = Some(insertedCampaign.id),
+                  status = AnnotationProjectStatus.Waiting
+                ),
+                parent
+              )
+            _ <- AnnotationProjectDao.update(
+              insertedProject.toProject
+                .copy(status = AnnotationProjectStatus.Ready),
+              insertedProject.id
+            )
+            campaignCopies <- children traverse { child =>
+              CampaignDao.copyCampaign(insertedCampaign.id, child)
+            }
+            projectCopies <- (campaignCopies traverse { c =>
+              AnnotationProjectDao.listByCampaign(c.id)
+            }) map (_.flatten)
+            projectCopiesWithRelated <- projectCopies traverse { pc =>
+              AnnotationProjectDao.getWithRelatedById(pc.id)
+            } map (_.flatten)
+            tasks <- TaskDao.insertTasks(
+              fixupTaskFeaturesCollection(
+                taskFeaturesCreate,
+                projectCopiesWithRelated.head
+              ),
+              parent
+            )
+            _ <- TaskDao.updateTask(
```

Comment:
What is the purpose of the `Labeled` `Label` task? it doesn't seem to have much to do with the assertion